src/platform_impl/android/ndk_glue.rs

```diff
@@ -54,7 +54,7 @@ macro_rules! android_binding {
   };
   ($domain:ident, $package:ident, $activity:ident, $setup:path, $main:ident, $tao:path) => {{
     // NOTE: be careful when changing how this use statement is written
-    use $tao::{platform::android::prelude::android_fn, platform::android::prelude::*};
+    use $tao::platform::android::prelude::{android_fn, *};
```

Comment:
please revert this change

Reply:
stupid muscle memory, i didn't want to do `git add .` 🙃 

is there an attribute like ts-ignore we can use to tell cargo not to format that?

Reply:
maybe yeah, but i have no idea what's going on 😂